### ISSUE TITLE
Kali Linux 2022.2 released

### ISSUE THREAD

**Issue by m-1-k-3** [e-m-b-a/embark#47]
Kali Linux 2022.2 released here: https://www.kali.org/blog/kali-linux-2022-2-release/ 
We need to test EMBArk on it

**Comment by torabi12** [comment]
Hello Benedikt,

I also tried the latest 2022.2 kali linux with EMBArk, I would share some findings.

I tested this firmware:
https://dlcdnets.asus.com/pub/ASUS/mb/BIOS/PRIME-B560M-K-ASUS-1601.zip

After uploading the CAP file I entered the details of the firmware like this:
![kép](https://user-images.githubusercontent.com/98665813/170856864-f009bd88-8c1e-4925-a0c9-6d7b695e16f6.png)

and ran the scan. I opened the final report in EMBArk and the numbers were missing on the final aggregator page:
![kép](https://user-images.githubusercontent.com/98665813/170856905-eaa5d772-eec5-4f6b-a1d7-d450d3a656fd.png)

The device and the additional notes fields are not the same as I have entered the original data before starting the scan. I think this is not a new issue I have experienced the same a few weeks ago also.

In my scan there is a stacs_pw_hashes page in the report, if I open it I got an error page:
![kép](https://user-images.githubusercontent.com/98665813/170856985-33e6ddc5-3249-443a-87dc-16cbc3e2f355.png)

I executed 2 different scans but the main page does now show any statistics about the results:
![kép](https://user-images.githubusercontent.com/98665813/170857018-2c1da4e9-b3ac-4406-a838-6b4a896a2f5e.png)

and I tried the delete function again, it was working, but the report page contains the deleted firmware which is not bad I think, but if I open the detailed view I get an almost empty page, only the entropy picture is shown:

![kép](https://user-images.githubusercontent.com/98665813/170857067-3c12215a-f3a9-4abf-8131-8ae4772c9121.png)

Used system:
https://kali.download/virtual-images/kali-2022.2/kali-linux-2022.2-vmware-amd64.7z
VMware Workstation Pro
I installed EMBArk yesterday: 05/28/2022

I didn't have any errors or issues during the installation.

Regards,
torabi

**Comment by BenediktMKuehne** [comment]
Hi torabi,

in order to get rid of the whole "direct user-input into root-shell"-problem, we will probably get rid of all user-input in the EMBA-command.
-> *Not a bug but intended behavior*
(for now it just gets rid of special chars)

The Attribute-Error is a bug I'll squash.

Did the EMBA process finish correctly?
You could check the `emba.log` file(s) (under /app/www/emba_logs/<analysis-id>)

**Comment by torabi12** [comment]
Hi,

I did a new installation but the folder is empty:
kali㉿kali)-[~/embark/www/emba_logs]
└─$ ll
total 0

I checked the content of the terminal where the installer was running and there was no errors or serious problems.

If I execute az emba scan there is some content but right after installation there was nothing in the folder.
I am going to do some new scans to see the behaviour of the system.

**Comment by torabi12** [comment]
I think the empty main page was a temporary issue, I did a new installation with the same releases tonight and after the 1st scan the page was OK:
![kép](https://user-images.githubusercontent.com/98665813/171054648-0cd8de0f-96f9-435d-a40b-38644e5f7687.png)

after this I deleted the scan/FW and on the details list I opened the details view and I got the same empty page with the entropy graph:
![kép](https://user-images.githubusercontent.com/98665813/171054808-dd59134c-7163-4127-8498-96d4150e2cb7.png)

If this page is available it should display the full content maybe.

Lastly the stacs page was also wrong like the error above:
http://embark.local/emba_logs/3fbd8afc-6076-4b00-8f62-2174f9ef4817/emba_logs/html-report//etc/stacs_pw_hashes.html
